src/Shell/Helper/TableHelper.php

```diff
@@ -78,6 +78,10 @@ protected function _rowSeparator($widths)
      */
     protected function _render($row, $widths, $options = [])
     {
+        if(empty($rows) || !is_array($row)) {
```

Comment:
Typo. `empty()` is not suitable to evaluate variables. Use `!$rows` instead. See #8397